build/dsc/templates/dsc_type_spec.rb.erb

```diff
@@ -44,7 +44,17 @@ describe Puppet::Type.type(:dsc_<%= resource.friendlyname.downcase %>) do
 <%    end # if property.required? -%>
 <%    if property.values -%>
 <%      property.values.each do |value| -%>
+<%    if property.array? -%>
+  it 'should accept dsc_<%= property.name.downcase %> predefined value <%= value %>' do
+    dsc_<%= resource.friendlyname.downcase %>[:dsc_<%= property.name.downcase %>] = '<%= value %>'
+    expect(dsc_<%= resource.friendlyname.downcase %>[:dsc_<%= property.name.downcase %>]).to eq(['<%= value %>'])
+  end
 
+  it 'should accept dsc_<%= property.name.downcase %> predefined value <%= value.downcase %>' do
+    dsc_<%= resource.friendlyname.downcase %>[:dsc_<%= property.name.downcase %>] = '<%= value.downcase %>'
+    expect(dsc_<%= resource.friendlyname.downcase %>[:dsc_<%= property.name.downcase %>]).to eq(['<%= value.downcase %>'])
+  end
+  <% else %>
```

Comment:
FYI we've got some whitespace noise being injected in to a number of files b/c of the 2 space indent here.